gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-common/src/main/java/io/gravitee/am/gateway/handler/common/email/impl/EmailServiceImpl.java

```diff
@@ -202,18 +202,31 @@ public void send(Email email) {
     private void sendEmail(Email email, User user, Client client) {
         try {
             final Locale preferredLanguage = preferredLanguage(user, Locale.ENGLISH);
+            final Map<String, Object> params = email.getParams();
+            final Email emailToSend = new Email(email);
+
+            // compute email - from
+            final Template fromTemplate = new Template("from", new StringReader(email.getFrom()), freemarkerConfiguration);
```

Comment:
suggestion: it looks like we've upgraded Freemarker to a version that has some convenience ctors, so we can skip conversions to StringReader in our code:

```suggestion
            final Template fromTemplate = new Template("from", email.getFrom(), freemarkerConfiguration);
```